pages/csoc/resources.vue

```diff
@@ -1,8 +1,8 @@
 <template>
-  <div>
+  <v-container fluid>
```

Comment:
@king-11  i don't think that resolved the issue completely , still some part of content is  over lapping

Reply:
I tried reloading the page several times didn't happen can you post an ss

Reply:
Just try navigating to blog then to csoc page .. then you can notice the issue

Reply:
![Screenshot_20201223-232106.jpg](https://user-images.githubusercontent.com/52061363/103024291-8e8c9480-4575-11eb-85e1-aca7407aab55.jpg)

The issue persists I guess with blog is the issue that it's 404 page whose contents aren't wrapped in v-main or container